### DISCUSSION TITLE
Nova vs enhanced model: which has better accuracy?

### DISCUSSION THREAD

**Discussion by pdjewell** [deepgram/community#218] [General help] [answered]
Hi there,
Really loving deepgram, so easy to use.

I was trying to work out which has better accuracy, the nova or the enhanced model, as it is not clear to me from the website? 
Do you have any accuracy (e.g. WER) scores on benchmarks comparing these two models?
I can run my own tests, but just wanted to see if you had the answer already. Thanks!

**Comment by shirgoldbird** [comment, marked as answer]
Hi @pdjewell! For nearly all use cases, Nova is our most accurate model, and our cheapest one too. You can view some WER stats on Nova vs competitors on [our blog](https://blog.deepgram.com/nova-speech-to-text-whisper-api/).

We always recommend running some tests with your own audio to see how Deepgram's different [tiers](https://developers.deepgram.com/docs/tier) and [models](https://developers.deepgram.com/docs/model) stack up on your data. Hope this helps!
